src/mongocxx/options/encrypt.hpp

```diff
@@ -103,11 +103,17 @@ class MONGOCXX_API encrypt {
     /// queryType only applies when algorithm is "Indexed" or "RangePreview".
     /// It is an error to set queryType when algorithm is not "Indexed" or "RangePreview".
     ///
+    /// Queryable Encryption is in Public Technical Preview. Queryable Encryption should not be used
+    /// in production and is subject to backwards breaking changes.
+    ///
     enum class encryption_query_type : std::uint8_t { k_equality };
 
     ///
     /// Sets the algorithm to use for encryption.
     ///
+    /// Queryable Encryption is in Public Technical Preview. Queryable Encryption should not be used
+    /// in production and is subject to backwards breaking changes.
+    ///
```

Comment:
Should this be moved to the `query_type()` member functions? It is not clear which item described in CXX-2523 motivates the additions to `algorithm()` member functions.

Reply:
Good point. I believe I should add this line above that statement (similar to the Go driver)

```
// Indexed and Unindexed are used for Queryable Encryption.
```

Reply:
To followup on recent commits, my question was whether `algorithm()` is in the set of entities to document with the "Public Technical Preview" notice, because `EncryptOpts.algorithm` does not appear to be listed in CXX-2523.

> AutoEncryptionOpts.encryptedFieldsMap
> AutoEncryptionOpts.bypassQueryAnalysis
> EncryptOpts.contentionFactor
> EncryptOpts.queryType
> QueryType enum

Reply:
For what it's worth, the Go driver has this documentation on the SetAlgorithm method: https://github.com/mongodb/mongo-go-driver/commit/d6ab9dd9ae8f2740b6cd42891ce484bf245113ed#diff-a7544f75aa4048518e5cddaca2f89e331e678ba8d494b3fa5a43f16d083487e4R52-R56

Reply:
@kevinAlbs Should the Downstream Changes Summary for DRIVERS-2349 be updated to include `EncryptOpts.algorithm` (Indexed and Unindexed) given implementations in (at least) the C and Go Drivers? Or is there something I am missing that already specifies the Indexed and Unindexed encryption algorithms should be included in the entities to document?

Reply:
> @kevinAlbs Should the Downstream Changes Summary for [DRIVERS-2349](https://jira.mongodb.org/browse/DRIVERS-2349) be updated to include `EncryptOpts.algorithm` (Indexed and Unindexed) given implementations in (at least) the C and Go Drivers? Or is there something I am missing that already specifies the Indexed and Unindexed encryption algorithms should be included in the entities to document?

Good catch. Updated the Downstream Changes Summary of DRIVERS-2349 to include:
> - The "Indexed" and "Unindexed" values for EncryptOpts.algorithm